packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.spec.js

```diff
@@ -138,14 +144,17 @@ describe('Component > ProjectAboutPage & Connector', function () {
       fireEvent.click(dropdown)
       const navContainer = getByTestId('mobile-about-pages-nav')
       const links = getAllByRole(navContainer, 'link')
-      expect(links).to.have.lengthOf(3)
+      expect(links).to.have.lengthOf(2)
```

Comment:
I don't think this test expects only 2 links. The mockStore provides a project with Research and Results pages. The Team page should also always be available in the browser, so the menu in this unit test should include 3 links (Research, Team, Results). Is something else going on here?

Reply:
You're right. I saw two pages in the mock, so changed this, but there should be three in the links array. Something else is going on here.

Reply:
I Fancy Cats, which doesn't have a team page, works in the browser so something weird is going on with the tests.

Reply:
I'd forgotten to change `page?.content` to `page?.strings?.content`. Should be fixed now.